ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -303,13 +303,72 @@
 {%                         set mode = "-" %}
 {%                     endif %}
 | {{ ethernet_interface }} | {{ channel_group }} | {{ isis_instance }} | {{ isis_metric }} | {{ mode }} | {{ isis_circuit_type }} |
-{%                endif %}
+{%                 endif %}
 {%             endif %}
 {%         endfor %}
 {%     endif %}
 {%     if port_channel_interfaces_isis | length > 0 %}
  *Inherited from Port-Channel Interface
 {%     endif %}
+{# EVPN Multihoming #}
+{%     set evpn_mh_ethernet_interfaces = [] %}
+{%     set evpn_dfe_ethernet_interfaces = [] %}
+{%     set evpn_mpls_ethernet_interfaces = [] %}
+{%     for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort %}
+{%         if ethernet_interfaces[ethernet_interface].evpn_ethernet_segment.esi is arista.avd.defined %}
+{%             do evpn_mh_ethernet_interfaces.append(ethernet_interface) %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].evpn_ethernet_segment.designated_forwarder_election is arista.avd.defined %}
+{%             do evpn_dfe_ethernet_interfaces.append(ethernet_interface) %}
+{%         endif %}
+{%         if ethernet_interfaces[ethernet_interface].evpn_ethernet_segment.mpls is arista.avd.defined %}
+{%             do evpn_mpls_ethernet_interfaces.append(ethernet_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if evpn_mh_ethernet_interfaces | length > 0 %}
+
+#### EVPN Multihoming
+
+##### EVPN Multihoming Summary
+
+| Interface | Ethernet Segment Identifier | Multihoming Redundancy Mode | Route Target |
+| --------- | --------------------------- | --------------------------- | ------------ |
+{%         for evpn_mh_ethernet_interface in evpn_mh_ethernet_interfaces %}
+{%             set esi = ethernet_interfaces[evpn_mh_ethernet_interface].evpn_ethernet_segment.esi %}
```

Comment:
this suggests that esi is required. the eos template has it as optional. What is correct?

Reply:
Basically, I am not documenting if esi isn't configured. But still, I am able to configure everything without esi. But that produces some trivial config. I will mention esi is required in README.